Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -463,26 +460,26 @@ private void UpdateInvalidLoginAttemptCount(User user, int newValue)
             {
                 user.Policy.InvalidLoginAttemptCount = newValue;
 
-                var maxCount = user.Policy.IsAdministrator ?
-                    3 :
-                    5;
+                var maxCount = user.Policy.IsAdministrator ? 3 : 5;
 
+                // TODO: Fix
+                /*
```

Comment:
why commenting? pls explain

Reply:
I mean, why did you decide to disable the code?

Reply:
Oh, it wasn't functional. It never executed.